src/auth-service/controllers/create-user.js

```diff
@@ -286,6 +286,64 @@ const createUser = {
       });
     }
   },
+  deleteMobileUserData: async (req, res) => {
+    try {
+      let responseFromDeleteAppData;
+      logText("We are deleting the app data.....");
+      const { userId } = req.query;
+      const { creationTime } = req.query;
+
+      let request = {};
+      request["body"] = {};
+      request["body"]["userId"] = userId;
+      request["body"]["creationTime"] = creationTime;
+      logObject("request:", request);
+     try {
+       responseFromDeleteAppData = await createUserUtil.deleteMobileUserData(
```

Comment:
@BenjaminSsempala 
please add the type keyword like `const` responseFromDeleteAppData =

Even though it is not required since JS is dynamically typed, it is good practice to do this.

Reply:
Thanks @Baalmart 
I've updated this